src/healthRecord/healthRecord.service.ts

```diff
@@ -90,4 +145,93 @@ export class HealthRecordService {
     const extractedColumns = [...new Map(allColumns.map((column) => [column[distinctiveKey], column])).values()];
```

Comment:
How is this different from the `allColumns`? Just curious

Reply:
It extracts all the unique value sets according to the specified columns